bigquery/query_places.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Places
+nav_order: 1
+parent: DC to BQ Sample Queries
+grand_parent: BigQuery
+---
+
+# Query Category: Places
+
+## Place containment
+
+* List all the places of type X contained in Y, e.g. counties in USA:
+
+```sql
+SELECT id AS PlaceId, name AS PlaceName
+FROM `data_commons.Place`
+WHERE EXISTS(SELECT * FROM UNNEST(all_types) AS T WHERE T = 'County') AND
+      EXISTS(SELECT * FROM UNNEST(linked_contained_in_place) AS C
+             WHERE C = 'country/USA')
+```
+
+* List all the places of type X neighboring Y:
```

Comment:
... within Z range of ...

(same query as the next one)

Could also add:

In this example, we use the "nearbyPlaces" triples to compute cities within 10KM distance of San Francisco.

Reply:
Done